.changeset/lemon-baboons-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"eppo_core": patch
+---
+
+Ingestion: Fix JSON serialization of Event type field
```

Comment:
This is going to appear in public release notes / changelog, so want to mark all mentions of event ingestion as "unstable"
```suggestion
[unstable] Event Ingestion: Fix JSON serialization of Event type field
```